NitroxServer/Communication/Packets/Processors/SetIntroCinematicModeProcessor.cs

```diff
@@ -0,0 +1,32 @@
+using System.Linq;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public class SetIntroCinematicModeProcessor : AuthenticatedPacketProcessor<SetIntroCinematicMode>
+{
+    private readonly PlayerManager playerManager;
+
+    public SetIntroCinematicModeProcessor(PlayerManager playerManager)
+    {
+        this.playerManager = playerManager;
+    }
+    public override void Process(SetIntroCinematicMode packet, Player player)
+    {
+        player.PlayerContext.IntroCinematicMode = packet.Mode;
```

Comment:
Probably want to modify the packet to make sure that the provided ``packet.PlayerId`` is equals to ``player.Id`` just in case (security-wise).